tests/testthat/test_base_generateFilterValuesData.R

```diff
@@ -143,3 +143,9 @@ test_that("filter method 'variance' works with missing values", {
   fi = generateFilterValuesData(regr.na.num.task, method = "variance")
   expect_false(anyMissing(fi$data$variance))
 })
+
+test_that("ensemble methods work", {
+  fi = generateFilterValuesData(multiclass.task, method = c('gain.ratio','information.gain'),
+                                ensemble.method = c("E-Borda", "E-min"))
```

Comment:
What does it mean if multiple ensemble methods are specified?

Reply:
The same as if you would use multiple single ones. You get back a DF with all listed rankings when you use `generateFilterValuesData()`.